src/toolbox.ts

```diff
@@ -1970,96 +2160,112 @@ export class KeypointSliderItem extends ToolboxItem {
+    constructor(ulabel: ULabel, kwargs: { [name: string]: any } = null) {
+        super();
+
+        this.ulabel = ulabel;
+
         // Get this component's config from ulabel's config
-        this.config = this.ulabel.config.distance_filter_toolbox_item
+        this.config = this.ulabel.config.distance_filter_toolbox_item;
 
         // For each key missing from the config, set the default value
         for (const key in DEFAULT_FILTER_DISTANCE_CONFIG) {
-            if (!this.config.hasOwnProperty(key)) {
-                this.config[key] = DEFAULT_FILTER_DISTANCE_CONFIG[key]
+            if (!Object.prototype.hasOwnProperty.call(this.config, key)) {
+                this.config[key] = DEFAULT_FILTER_DISTANCE_CONFIG[key];
             }
         }
 
         // Set the component's properties to be the same as the config's properties
         for (const property in this.config) {
-            this[property] = this.config[property]
+            this[property] = this.config[property];
         }
 
         // Force disable multi-class mode if the config doesn't allow it
-        if (this.disable_multi_class_mode) this.multi_class_mode = false
-        
+        if (this.disable_multi_class_mode) this.multi_class_mode = false;
+
         // Get if the options should be collapsed from local storage
-        this.collapse_options = get_local_storage_item("filterDistanceCollapseOptions")
- 
+        this.collapse_options = get_local_storage_item(
+            "filterDistanceCollapseOptions",
+        );
+
         // Create an overlay and determine whether or not it should be displayed
-        this.create_overlay()
+        this.create_overlay();
 
         // Check if localStorage has a value for showing the overlay
-        const show_overlay = get_local_storage_item("filterDistanceShowOverlay")
+        const show_overlay = get_local_storage_item(
+            "filterDistanceShowOverlay",
+        );
         // Guard against null values
-        this.show_overlay = show_overlay !== null ? show_overlay : this.show_overlay
+        this.show_overlay =
+            show_overlay !== null ? show_overlay : this.show_overlay;
         this.overlay.update_display_overlay(this.show_overlay);
 
         // Check if localStorage has a value for filtering during polyline move
-        const filter_during_polyline_move = get_local_storage_item("filterDistanceFilterDuringPolylineMove")
+        const filter_during_polyline_move = get_local_storage_item(
+            "filterDistanceFilterDuringPolylineMove",
+        );
         // Guard against null values
-        this.filter_during_polyline_move = filter_during_polyline_move !== null ? filter_during_polyline_move : this.filter_during_polyline_move
+        this.filter_during_polyline_move =
+            filter_during_polyline_move !== null
+                ? filter_during_polyline_move
+                : this.filter_during_polyline_move;
```

Comment:
Same concern as [this comment](https://github.com/SenteraLLC/ulabel/pull/175/files#r1781279185).